ci/jira_query.py

```diff
@@ -22,15 +24,18 @@
 # Expected components from the commit message, and directory in src/, src/client or utils/ is also
 # valid.  We've never checked/enforced these before so there have been a lot of values used in the
 # past.
-VALID_COMPONENTS = ('build', 'ci', 'doc', 'gha', 'test')
+VALID_COMPONENTS = ('build', 'ci', 'doc', 'gha', 'il', 'mercury', 'test')
```

Comment:
I could add a bunch here

vos, bio, vea, object, common, gurt, cart to name a few that I've used. I can push those as needed, I suppose.

Reply:
Read the comment.  And subdirectory of src/ src/client or utils/ is also allowed.

Reply:
ah, missed that :-)